api-tests/src/utils/jar-generator.ts

```diff
@@ -30,6 +31,14 @@ export const generateJarPayload = async (
     ),
   ) as JarRequest;
 
+  payloadData.iss = ORCH_CLIENT || "orchStub";
+  payloadData.client_id = ORCH_CLIENT || "orchStub";
+
+  if (session.journeyType === "reverification") {
+    payloadData.iss = AUTH_CLIENT || "authStub";
+    payloadData.client_id = AUTH_CLIENT || "authStub";
+  }
```

Comment:
Do we still need all this stuff with ORCH_CLIENT and AUTH_CLIENT?

Reply:
ah 🤦🏻‍♀️ thanks for the spot